cabal-install/Distribution/Client/IndexUtils.hs

```diff
@@ -445,12 +522,14 @@ is01Index (SandboxIndex _)   = False
 
 updatePackageIndexCacheFile :: Verbosity -> Index -> IO ()
 updatePackageIndexCacheFile verbosity index = do
-    info verbosity ("Updating index cache file " ++ cacheFile index)
+    info verbosity ("Updating index cache file " ++ cacheFile index ++ " ...")
     withIndexEntries index $ \entries -> do
       let cache = Cache { cacheEntries = entries
                         , cacheHeadTs  = maxTs entries
                         }
       writeIndexCache index cache
+      notice verbosity ("Index cache updated to index-state "
+                        ++ display (cacheHeadTs cache))
```

Comment:
Is this really to be a user facing message? If so we don't want to talk about internal things like caches. Rather we'd want the overall output to be something like

    Downloading the latest package index from hackage.haskell.org
    Updated to index state 2017-12-31T23:59:59Z

and so we'd want to write that notice in the same place we do the "downloading" bit, also since these things happen concurrently so the order of the notices matters when there's multiple repos.

My advice: swap this to be `info` for now, rather than `notice`